main.go

```diff
@@ -92,6 +93,7 @@ func init() {
 	utilruntime.Must(clusterv1alpha1.AddToScheme(scheme))
 	utilruntime.Must(operatorsv1alpha1.AddToScheme(scheme))
 	utilruntime.Must(nadscheme.AddToScheme(scheme))
+	utilruntime.Must(networkingv1.AddToScheme(scheme))
```

Comment:
As of prev comments, I question the need

Reply:
Answered here [comment](https://github.com/red-hat-storage/ocs-operator/pull/2465#issuecomment-2014332087)

Reply:
Remove network policy for prometheus
see [comment](https://github.com/red-hat-storage/ocs-operator/pull/2465#issuecomment-2014742465)